src/main/java/org/stellar/base/ManagerOfferOperation.java

```diff
@@ -52,15 +50,8 @@ public long getOfferId() {
     Int64 amount = new Int64();
     amount.setint64(Long.valueOf(mAmount));
     op.setamount(amount);
-    RationalApproximation rationalApproximation = toRationalApproximation(mPrice);
-    Price price = new Price();
-    Int32 n = new Int32();
-    n.setint32((int) rationalApproximation.a);
-    Int32 d = new Int32();
-    d.setint32((int) rationalApproximation.b);
-    price.setn(n);
-    price.setd(d);
-    op.setprice(price);
+    Price price = Price.rationalApproximation(mPrice);
```

Comment:
I don't think we need to have this named `rationalApproximation`.  We don't need to communicate the implementation of how we convert a double to a price, as it may change.

I prefer:  `Price.fromDouble(mPrice)`